src/components/CustomerQuota/TransactionsGroup.tsx

```diff
@@ -33,6 +33,7 @@ export const styles = StyleSheet.create({
   },
   itemDetail: {
     fontSize: fontSize(-1),
+    marginRight: size(3),
```

Comment:
curious, what's the reason for only setting `marginRight` here, while all margins are set for storybook [here](https://github.com/rationally-app/mobile-application/pull/505/files#diff-5de6e9e2a567e09c506226cea2bc7489790770071d9fb6ed0084cc20f8de0104R140-R141)?

Reply:
The View margin of 3 is to create space “outside” CheckoutSuccessCard wrt phone view. For this overflow, ‘itemDetailBorder' style of marginLeft and marginRight result in 'itemDetail’ component to shift to the right. Thus, added marginRight in 'itemDetail' to compensate the shift wrt CheckoutSuccessCard.